### PR TITLE
Ds 500 investigate solution on front end to handle headings of nested and non nested articles

### PR DESCRIPTION
**Pull request by chesterroberts-vs**
Updated AccordionItem, Article, ModuleWrapper and StyledList to include headingLevel prop that overrides the default level of the heading. This will be used when the component is used within another component.

The StyleList component has had the heading and intro section removed and will use the ModuleWrapper component instead.

The ModuleWrapper component has had a BusinessSupport prop added which sets the text alignment and the column sizes.

### CODE REVIEW

File: src/components/module-wrapper/ModuleWrapper.vue [hunk 1]
```diff
@@ -76,6 +70,32 @@ export default {
         VsCol,
     },
     props: {
+        /**
+         * AnchorId will be used to set an id on the heading,
+         * allowing for anchor links (ToC)
+         */
+        anchorId: {
```
Comment (GemmaLouise) [inline src/components/module-wrapper/ModuleWrapper.vue:77]: This new prop is missing unit tests

File: src/components/accordion/components/__tests__/AccordionItem.jest.spec.js [hunk 2]
```diff
@@ -82,6 +82,30 @@ describe('VsAccordionItem', () => {
 
             expect(badge.exists()).toBe(true);
         });
+
+        it(':headingLevel - changes the heading to the corresponding level', async() => {
+            const wrapper = factoryMount({
+                slots: {
+                    'vs-module-wrapper-heading': 'Module wrapper heading',
```
[the remark this answers is not in the document]
Comment (duncansanderson) [reply src/components/accordion/components/__tests__/AccordionItem.jest.spec.js]: I've updated this

File: src/components/module-wrapper/ModuleWrapper.vue (hunk 1 above)
Comment (duncansanderson) [reply src/components/module-wrapper/ModuleWrapper.vue:77]: I've added a test for the anchorId prop

File: src/components/accordion/components/AccordionItem.vue [hunk 3]
```diff
@@ -8,6 +8,7 @@
         <BCardHeader
             class="vs-accordion-item__card-header"
             data-test="vs-accordion__item-header"
+            data-bob="test"
```
Comment (GemmaLouise) [inline src/components/accordion/components/AccordionItem.vue:11]: Should this be removed?

**Review by GemmaLouise — APPROVED** [review]

**Review by mjpvs — APPROVED** [review]

**Pull request merged**